lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -187,9 +231,8 @@ export default function FilePickerApp({ onSubmit }) {
         )}
         {content && (
           <FilePickerFormFields
-            ltiLaunchURL={ltiLaunchUrl}
             content={content}
-            formFields={formFields}
+            formFields={{ ...formFields, ...deeplinkingFields }}
```

Comment:
`formFields` could be merged into deeplinkingFields, I can't see a reason to have both.

Reply:
To check my understanding here, `formFields` are static fields which don't depend on the selected content, whereas `deeplinkingFields` are the dynamic fields that do depend on the content?

Reply:
I think that distincit might be true currently but at the end they are all (in this PR) transparent values that the frontend forwards back to the backend. If the backend want to separate them in any way I think it should do that on the "data" property of the new API .

Reply:
Yeah that might be one distinction between the two.